tasks/main.yml

```diff
@@ -121,3 +121,6 @@
     creates: "/etc/sysconfig/iptables"
 #  changed_when: False
   when: ansible_pkg_mgr == "yum"
+
+- include: fail2ban.yml
+  when: ansible_default_ipv4.address | ipaddr('10.0.0.0/8') or ansible_default_ipv4.address | ipaddr('172.16.0.0/12') or ansible_default_ipv4.address | ipaddr('192.168.0.0/16')
```

Comment:
fail2ban should be installed when default ip address is NOT in subnets defined in RFC1918.